test-clients/src/main/java/com/hedera/services/bdd/spec/HapiApiSpec.java

```diff
@@ -485,7 +485,7 @@ public static void runInCiMode(
 			}};
 			final var explicitCiProps = MapPropertySource.parsedFromCommaDelimited(ciPropertiesMap);
 			if (explicitCiProps.has(CI_PROPS_FLAG_FOR_NO_UNRECOVERABLE_NETWORK_FAILURES)) {
-				ciPropsSource.put("warnings.suppressUnknowns", "true");
+				ciPropsSource.put("warnings.suppressUnrecoverableNetworkFailures", "true");
```

Comment:
Where is this `warnings.suppressUnrecoverableNetworkFailures`? I couldn't find it on my local.

Reply:
nvm, I was on a different branch. Should be looking into the `master-plus-sdk-0.15.0-alpha.1` branch.